### ISSUE TITLE
Action fails if `no-certificate` is not defined in v2.5.0

### ISSUE THREAD

**Issue by GlitchWitch** [bakerkretzmar/laravel-deploy-preview#39]
# Problem
When using v2.5.0 with both `existing-certificate` and `existing-certificate-key` defined, but `no-certificate` not defined, the action errors with the following message:

```
Error: Input does not meet YAML 1.2 "Core Schema" specification: no-certificate
Support boolean input list: `true | True | TRUE | false | False | FALSE`
```

Example config (working in previous versions):

```yaml
[...omitted for brevity...]
        with:
          forge-token: ${{ secrets.FORGE_TOKEN }}
          servers: |
            ${{ secrets.FORGE_PREVIEW_SERVERS }}
          existing-certificate: |
            ${{ secrets.SSL_CERTIFICATE }}
          existing-certificate-key: |
            ${{ secrets.SSL_PRIVATE_KEY }}
          after-deploy: |
            $FORGE_PHP artisan down
[...omitted for brevity...]
```

# Solution

While not an ideal fix, setting `no-certificate` to `false` seems to fix the error presented.

**Comment by bakerkretzmar** [comment]
Whoops thanks, will get this fixed today!

**Comment by bakerkretzmar** [comment]
Should be fixed in v2.5.1!
